cms/djangoapps/contentstore/management/commands/add_test_transcript.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
```

Comment:
Where are we using this management command? Also do we need to write unit tests for this?

Reply:
That's just a test command, we will be removing it.